native/cocos/audio/android/audio_utils/include/audio_utils/format.h

```diff
@@ -18,10 +18,15 @@
 #define COCOS_AUDIO_FORMAT_H
 
 #include <stdint.h>
+#if CC_PLATFORM == CC_PLATFORM_ANDROID
 #include <sys/cdefs.h>
+#endif
+
 #include "audio/android/audio.h"
 
-__BEGIN_DECLS
+#ifdef	__cplusplus
+extern "C" {
```

Comment:
No need to check __cplusplus, it is definitly extern "C"

Reply:
It's better to add it, because it includes both c and C++ files, and it should be safer to add this